app.rb

```diff
@@ -37,6 +38,27 @@ def config
       @config ||= Config.new
     end
 
+    get '/attempt-events' do
+      auth_token = 'abc123'
```

Comment:
Should match [one of these values](https://github.com/n1zyy/identity-idp/pull/1/files#r921478004)